### PR TITLE
Complete the homework №1

### PR DESCRIPTION
**Pull request by natak8200**

### CODE REVIEW

File: src/main/java/hw_1/Task1.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package hw_1;
+
+public class Task1 {
+    public static void main(String[] args) {
+
+    String name = " Nataliya";
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task1.java]: Extra space. Should be not " Nataliya" but "Nataliya"

File: src/main/java/hw_1/Task1.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    int age = 35;
+    float weight = 50.5f;
+    String result = name +" "+ age+" " + weight;
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task1.java]: Should be not "name +" "+ age+" " + weight;" but "name + " " + age + " " + weight";

File: src/main/java/hw_1/Task1.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        System.out.println(result );
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task1.java]: Extra tabulation.
Extra space after result word

File: src/main/java/hw_1/Task2.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package hw_1;
+
+public class Task2 {
+    public static void main(String[] args) {
+
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task2.java]: Extra empty line/ Please remove

File: src/main/java/hw_1/Task2.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        int a = 1;
+        int b = 2;
+        System.out.println(a);
+        System.out.println(b);
+
+       int  c = a;
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task2.java]: Let's name it helper but not "c"

File: src/main/java/hw_1/Task3.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
+package hw_1;public class Task3 {
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task3.java]: "public class Task3" should be from the new line

File: src/main/java/hw_1/Task3.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 3;
+
+        System.out.println(a);
+        System.out.println(b);
+
+        a = a + b;
+        b = a -b;
+        a = a-b;
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task3.java]: should be extra space after "a" and after "-".
On the 10 row the same

File: src/main/java/hw_1/Task1.java [hunk 8]
```diff
@@ -0,0 +1,13 @@
+package hw_1;
+
+public class Task1 {
+    public static void main(String[] args) {
+
+    String name = "Nataliya";
+    int age = 35;
+    float weight = 50.5f;
+    String result = name + " " + age+ " " + weight;
```
Comment (natak8200) [inline src/main/java/hw_1/Task1.java]: fixed

File: src/main/java/hw_1/Task1.java (hunk 3 above)
Comment (natak8200) [reply src/main/java/hw_1/Task1.java]: fixed

File: src/main/java/hw_1/Task2.java (hunk 4 above)
Comment (natak8200) [reply src/main/java/hw_1/Task2.java]: fixed

File: src/main/java/hw_1/Task1.java (hunk 2 above)
Comment (natak8200) [reply src/main/java/hw_1/Task1.java]: fixed

**Pull request merged**